RELEASE/scripts/autoscend/combat/auto_combat_default_stage5.ash

```diff
@@ -335,6 +331,95 @@ string auto_combatDefaultStage5(int round, monster enemy, string text)
 			costMajor = mp_cost($skill[Heroic Belch]);
 		}
 		break;
+	
+	case $class[Avatar of Jarlsberg]:
```

Comment:
Not saying you're wrong to do so, but is there a reason for merging jarlsberg combat back into the default killing section (stage5) rather than having its own file?

Reply:
because there is other combat logic at the bottom of this file to decide which of attack major or minor to use